org.flatpak.am-okay.yml

```diff
@@ -0,0 +1,81 @@
+id: org.flatpak.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --share=network
+    - --socket=x11
+    - --filesystem=host
+    - --filesystem=/tmp
+    - --device=all
+    - --filesystem=/media
+
+modules:
+    - name: am-okay-package
+      buildsystem: simple      
+      sources:
+          - type: git 
+            url: https://github.com/baldeuniversel/am-okay.git 
+            commit: ed3e921ec1241c237887aa9041cad8ac9f0ee3b0 
+
+      build-commands: 
+            - install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/bin/am-okay /app/bin/am-okay
+
+            - install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/share/metainfo/io.github.baldeuniversel.am-okay.metainfo.xml \
+              /app/share/metainfo
+            
+            - |
+              install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/usr/share/icons/hicolor/128x128/apps/org.flatpak.am-okay.png \
+              /app/usr/share/icons/hicolor/128x128/apps
+
+            - |
+              install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/usr/local/share/screenshot/screenshot_scenario_1_am-okay.png \
```

Comment:
There's not need to install this screenshot file.

Reply:
While I do this `org.flatpak.am_okay` the command line will be `am_okay` (during the test) .

Reply:
Sorry, I don't understand what you mean. And I don't understand how that relates to not installing a screenshot file.

Reply:
I wanted to say this -> 
When I build the package with the id `org.flatpak.am-okay`, the name of the program was `am_okay` instead of `am-okay` .

Reply:
I still don't get your point. You're not supposed to be using that ID (`org.flatpak.am-okay`) anyway.

If you don't want it to be named `am_okay`, simply choose the ID I mentioned [in a different thread](https://github.com/flathub/flathub/pull/5013#discussion_r1513513277).

Reply:
Okay , I'm going to change this now ( am-okay -> am_okay).

Reply:
Or you can simply use `io.github.baldeuniversel.am-okay`, which will keep it consistent with your app's name.

Reply:
Can I do this ( io.github.baldeuniversel.am-okay ) in both  `*.xml` and `*.yml` files  at the level of the `id` entry ?

Reply:
Yes, you can.

I suggested using underscore before because I thought Flatpak wouldn't allow the dash character, but it does allow it on the last component.

Reply:
Done !